### PR TITLE
Provide spikeReport getEndTime and supportsBackwardSeek

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

**Comment by hernando** [comment]
Needs SHA update once Brion PR is done

**Comment by hernando** [comment]
Ready to go, only SHA1 update missing.

**Comment by hernando** [comment]
ping

File: monsteer/plugin/spikeReport.cpp [hunk 1]
```diff
@@ -113,6 +115,12 @@ brion::Spikes SpikeReport::read(float min)
 {
     brion::Spikes spikes;
 
+    // First empty the buffered data with no timeout no matter the timestamps
+    // received. This gives the opportunity to update the end time without
+    // having to use a minimum timestamp that may cause the client to block.
+    while( _subscriber->receive( 0 ))
```
Comment (kaabimg) [inline monsteer/plugin/spikeReport.cpp]: Shouldn't this be added to readUntil and readSeek as well ?

Comment (hernando) [inline monsteer/plugin/spikeReport.cpp]: Good question. It's not absolutely necessary, but maybe it's a good idea for consistency.

**Comment by hernando** [comment]
Retest this please.

**Pull request merged**